src/pages/repository.html

```diff
@@ -13,8 +13,8 @@ <h1>{{pageTitle}}</h1>
             <form id="newRepoForm" class="form-horizontal" novalidate="novalidate">
                 <fieldset>
                     <div class="form-group row">
-                        <label for="id" class="col-xs-3 col-sm-3 col-md-2 col-form-label">Repository ID*</label>
-                        <div class="col-xs-4 col-sm-4 col-md-4" tooltip="{{repoTooltips.id}}">
+                        <label for="id" class="col-md-3 col-lg-2 col-form-label">Repository ID*</label>
+                        <div class="col-md-8 col-lg-7" tooltip="{{repoTooltips.id}}">
```

Comment:
I think it will look better if all inputs are the same length `col-lg-5`.
The repository name and description would still be visually separated from the rest by the indent.

Reply:
As a compromise this is now col-md-7 col-lg-6 col-xl-5.